### ISSUE TITLE
Cannot find module '/app/node_modules/grpc/src/node/extension_binary/grpc_node.node

### ISSUE THREAD

**Issue by dzoba** [grpc/grpc#10527]
Suddenly getting this error after the most recent update on npm:

```
2017-04-07T22:57:20.851841+00:00 app[web.1]: 7 Apr 22:57:20 - uncaughtException: 22f3a8c18c4c4661992478192a287402
2017-04-07T22:57:21.042842+00:00 app[web.1]: └── Error: Cannot find module '/app/node_modules/grpc/src/node/extension_binary/grpc_node.node
```

**Comment by prescottprue** [comment]
Seeing this too. It breaking our builds, and rolling back version does not help.

**Comment by dzoba** [comment]
I was able to avoid this issue by using npm instead of yarn to install dependencies.

**Comment by murgatroid99** [comment]
This looks like the result of a maybe-fixed yarn bug: yarnpkg/yarn#853.
